apps/admin_app/lib/admin_app_web/templates/option_type/index.html.eex

```diff
@@ -4,8 +4,8 @@
       <h2>Option Types</h2>
     </div>
     <div class="col-2 p-0 float-right text-right">
-      <%= link(" Add a type", to: option_type_path(@conn, :new), 
-        class: "btn btn-primary float-right fas fa-plus")%>
+      <%= link(raw("<i class='fas fa-plus'></i> Add a type"), to: option_type_path(@conn, :new), 
```

Comment:
Change the button text to `Add an Option Type`

Reply:
Done